src/models/neural_networks/linear_network.py

```diff
@@ -29,6 +30,7 @@ def __init__(self, layer_sizes: Union[int, List[int]],
                          include_latlons, include_monthly_aggs, include_yearly_aggs,
                          surrounding_pixels, ignore_vars, include_static)
 
+        self.input_layer_sizes = copy(layer_sizes)
         if type(layer_sizes) is int:
```

Comment:
Does this need to be done BEFORE L33 so that `input_layer_sizes` is also cast to a `List[int]` instead of being an `int`

Reply:
The idea of `input_layer_sizes` is to exactly mirror what the input to the class was, so that it can easily be loaded again.